src/apps/Settings/Settings.js

```diff
@@ -90,7 +106,16 @@ class Settings extends React.Component {
       selectedCurrency,
     } = this.state
     return (
-      <AppLayout title="Settings">
+      <AppLayout
```

Comment:
Maybe we should start using `AppView` from aragonUI instead, this would also let us avoid the awkward padding fixes for the title